### DISCUSSION TITLE
[question] VSCode Extension For Restarting Vue Language Server

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vuejs/language-tools#3498] [Q&A]
Hi.

Is there a VSCode extension, or an option in Volar extension, to have a button to restart Vue language server in the status bar?

For restarting TypeScript language server I simply click this icon in the status bar:

![image](https://github.com/vuejs/language-tools/assets/10191085/ae477852-f128-40e5-b7fc-05d13df39dbe)

**Comment by so1ve** [comment]
Currently: No. But this is likely a possible enhancement right? You can create a new issue for it.
